docs/ANDROID.md

```diff
@@ -1,3 +1,5 @@
+| Updated 26.02.2023 | Languages: EN, [CZ](/docs/lang/cs/ANDROID.md) |
```

Comment:
Please don't change the update dates if it's the only change - it'll make other translations look older than they actually are.